src/kOS/AddOns/Trajectories/Addon.cs

```diff
@@ -54,23 +83,30 @@ private BooleanValue IsVerTwoTwo()
             throw new KOSUnavailableAddonException("ISVERTWOTWO", "Trajectories");
         }
 
+        private BooleanValue IsVerTwoFour()
+        {
+            if (Available())
+                return TRWrapper.IsVerTwoFour;
+            throw new KOSUnavailableAddonException("ISVERTWOFOUR", "Trajectories");
+        }
+
         // Older suffixes.
         private GeoCoordinates ImpactPos()
         {
             if (shared.Vessel != FlightGlobals.ActiveVessel)
-                throw new KOSException("You may only call addons:TR:ImpactPos from the active vessel. Always check addons:tr:HasImpact");
+                throw new KOSException("You may only call addons:tr:IMPACTPOS from the active vessel. Always check addons:tr:HASIMPACT");
             if (Available())
             {
                 CelestialBody body = shared.Vessel.orbit.referenceBody;
                 Vector3? impactVect = TRWrapper.ImpactVector();
                 if (impactVect != null)
                 {
-                    var worldImpactPos = (Vector3d)impactVect + body.position;
-                    var lat = body.GetLatitude(worldImpactPos);
-                    var lng = Utils.DegreeFix(body.GetLongitude(worldImpactPos), -180);
+                    Vector3d worldImpactPos = (Vector3d)impactVect + body.position;
+                    double lat = body.GetLatitude(worldImpactPos);
+                    double lng = Utils.DegreeFix(body.GetLongitude(worldImpactPos), -180);
```

Comment:
BTW, thank you for this change.
The "var, go look up method elsewhere to see what type it really is" design pattern is one of the most annoying misfeatures of C#.  I call it "IDE-think" - people think "well, the IDE tells me the type so I don't have to state it", which is a PITA when reading it somewhere other than an IDE, like on github diff output.

Reply:
The only place I consider it good is when calling a constructor, because there the constructor's name IS the type so it's already clear from that.

Reply:
I have to agree, I rarely use var, usually only when throwing code together quick, once I'm happy with how the code works I will change the var's to the actual types.